plugins/stack-overflow/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@backstage/plugin-stack-overflow",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli package start",
+    "start": "backstage-cli package build",
+    "lint": "backstage-cli package lint",
+    "test": "backstage-cli package test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli package prepack",
+    "postpack": "backstage-cli package postpack",
+    "clean": "backstage-cli package clean"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.9.1",
+    "@backstage/plugin-home": "^0.4.18",
+    "@backstage/core-plugin-api": "^0.8.0",
+    "@backstage/search-common": "^0.3.1",
+    "@backstage/config": "^0.1.15",
+    "@backstage/theme": "^0.2.15",
+    "@material-ui/core": "^4.12.2",
+    "@material-ui/icons": "^4.9.1",
+    "qs": "^6.10.3",
+    "cross-fetch": "^3.1.5",
+    "lodash": "^4.17.21",
+    "react-use": "^17.2.4",
+    "@testing-library/jest-dom": "^5.10.1"
+  },
+  "peerDependencies": {
+    "@types/react": "^16.13.1 || ^17.0.0",
+    "react": "^16.13.1 || ^17.0.0"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.15.2",
+    "@backstage/test-utils": "^0.3.0",
+    "@testing-library/react": "^12.1.3"
+  },
+  "files": [
+    "dist"
```

Comment:
neither the frontend nor the backend seem to declare a config schema right? at least one of them probably should